controllers/ctltools.js

```diff
@@ -302,9 +322,24 @@ const getDatasheet = (req,res,next) => {
     }
 }
 
+const getMediumMedataHandler = (req,res,next) => {
+    const format = req.params.format;
+    const mediaSourceId = encodeURIComponent(req.params.mediaSourceId);
+    const mediumId = encodeURIComponent(req.params.mediumId);
+
+    try {
+        api(req, { version: 'v3' }).get(`/tools/external-tools/medium/${mediumId}/media-source/${format}/${mediaSourceId}/metadata`).then((reponse) => {
```

Comment:
We will have to talk to steliyan for the final url. After that you can also change the router.get url of this controller.